### PR TITLE
6-single-review view

### PR DESCRIPTION
**Pull request by ahmedmujtba**

### CODE REVIEW

File: src/components/ReviewPage.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { useParams } from "react-router-dom";
+
+export default function ReviewPage() {
+  const [singleReview, setReview] = useState({});
+  const [loading, setLoading] = useState(true);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    getSingleReview(review_id).then((review) => {
+      console.log(review);
```
Comment (CatHoang) [inline src/components/ReviewPage.jsx:12]: Please remove console logs when you have finished with them

File: src/components/ReviewPage.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            alt={singleReview.review.review_img_url}
+          ></img>
+          <h3 className="boldANDunderlined">
+            Title: {singleReview.review.title}
+          </h3>
+          <h3>Designer: {singleReview.review.designer}</h3>
+          <h3>Owner: {singleReview.review.owner}</h3>
+          <p>Review: {singleReview.review.review_body}</p>
+          <p>Category: {singleReview.review.category}</p>
+          <p>Created at: {singleReview.review.created_at}</p>
```
Comment (CatHoang) [inline src/components/ReviewPage.jsx]: I would think about how you can format the date so it is more readable for users

**Review by CatHoang — APPROVED** [review]
Hi Ahmed, looks good. I have left some minor feedback, I would also think about the responsiveness of your app as when it is mobile size - it cuts a-lot of your app off. Feel free to merge.

File: src/components/ReviewPage.jsx (hunk 2 above)
Comment (ahmedmujtba) [reply src/components/ReviewPage.jsx]: Thanks i'll have a look at that :)

**Pull request merged**